script/copy-debug-symbols.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import os
+import sys
+
+from lib.config import LINUX_BINARIES_TO_STRIP, PLATFORM
+from lib.util import execute, get_out_dir
+
+# It has to be done before stripping the binaries.
+def copy_debug_from_binaries(directory, target_cpu, compress):
+  for binary in LINUX_BINARIES_TO_STRIP:
+    binary_path = os.path.join(directory, binary)
+    if os.path.isfile(binary_path):
+      copy_debug_from_binary(binary_path, target_cpu, compress)
+
+def copy_debug_from_binary(binary_path, target_cpu, compress):
+  objcopy = get_objcopy_path(target_cpu)
+  if not objcopy:
+    return
+  debug_name = get_debug_name(binary_path)
+  cmd = [objcopy, '--only-keep-debug']
+  if compress:
+      cmd.extend(['--compress-debug-sections'])
+  cmd.extend([binary_path, debug_name])
+  execute(cmd)
+  return debug_name
+
+def add_debug_link_into_binaries(directory, target_cpu):
+  for binary in LINUX_BINARIES_TO_STRIP:
+    binary_path = os.path.join(directory, binary)
+    if os.path.isfile(binary_path):
+      add_debug_link_into_binary(binary_path, target_cpu)
+
+def add_debug_link_into_binary(binary_path, target_cpu):
+  objcopy = get_objcopy_path(target_cpu)
+  if not objcopy:
+    return
+  debug_name = get_debug_name(binary_path)
+  cmd = [objcopy, '--add-gnu-debuglink=' + debug_name, binary_path]
+  execute(cmd)
+
+def get_debug_name(binary_path):
+  return os.path.basename(binary_path) + '.debug'
+
+def get_objcopy_path(target_cpu):
+  if PLATFORM == 'linux' and target_cpu == 'x64':
+    return 'objcopy'
```

Comment:
Use the `objcopy` from the version of binutils that Chromium downloads at `src/third_party/binutils/Linux_x64/Release/bin/objcopy`

Reply:
Done